Implement the GitHub PR Review AI Assistant (#262)
fetchai/uAgents: integrations/github-pr-review-ai-assistant/README.md

## Overview

The GitHub PR Review AI Assistant is a AI agent designed to streamline the process of reviewing pull requests on GitHub.
Develop using fetch.ai agents and integrated with langchain and codedog, this AI Agent utilizes OpenAI's powerful LLM models to provide in-depth, automated reviews that improve code quality and collaboration efficiency.

## Features

- **Automated PR Reviews**: Quickly receive feedback on pull requests to improve code quality. 🚀
- **AI-Powered Insights**: Leverage GPT-3.5 and GPT-4 models for detailed code analysis. 🔍
- **Public Repository Support**: Currently supports reviewing pull requests on public GitHub repositories. 📖

## How to Use

To leverage the GitHub PR Review AI Assistant through DeltaV, follow these simple steps:

1. **Access DeltaV**:
   - Go to [DeltaV](https://deltav.agentverse.ai/).
   - Select the "All Service Group."
2. **Initiate a Request**:
   - Use the sample prompt: "I need help in reviewing a Github PR."
3. **Provide Required Information**:
   - **Repository URL**: When prompted, enter the public GitHub repository URL where your pull request is located. Ensure it is a public repository for accessibility.

>    - **Pull Request Number**: Specify the number of the pull request you need reviewed.

Reviewer: - **PR URL**: When prompted, enter the whole PR URL. At this point only **\*public\*** GitHub repositories are supported.

>    - **Repository URL**: When prompted, enter the public GitHub repository URL where your pull request is located. Ensure it is a public repository for accessibility.
>    - **Pull Request Number**: Specify the number of the pull request you need reviewed.

Reviewer: ```suggestion
   - **PR URL**: When prompted, enter the whole PR URL. At this point only **\*public\*** GitHub repositories are supported.
```
